resources/views/front/campaigns.blade.php

```diff
@@ -72,9 +72,11 @@
                 @else
                 <div class="row">
                     @foreach ($campaigns as $camp)
-                        <div class="col-sm-6">
-                            @include('front._campaign', ['campaign' => $camp, 'featured' => false])
-                        </div>
+                        @if (!$camp->isHidden())
```

Comment:
This check should be done in the database query, not in the view. Otherwise we'll have pages with less than 15 campaigns visible, and have a wrong pagination count.